include/boost/ut.hpp

```diff
@@ -237,6 +237,7 @@ template <class T>
   return t < T{} ? -t : t;
 }
 
+#undef min //prevents issues with min macros conflicts with this method declaration
```

Comment:
That's why macros are evil :disappointed: but I don't think we can simply undef it as that may break users who include both headers (ut being the last) and expect min to be there. Mor robust solution would be to simply change the `min` name to something like `min_value` :thinking: This way end-users who expect min to be there won't be affected after including ut.hpp.

Reply:
Arg I hate macros. It's one of the reasons why I liked UT so much. Anyways I made the change you recommended. It makes more sense.